README.md

```diff
@@ -55,6 +55,10 @@ Or, if you\'re using FreeBSD:
 
     # pkg install py36-pipenv
 
+When none of the above is an option(you need to use Python 3.6+):
```

Comment:
What is the reason Python 3.6+ is required? Since pipenv officially supports Python 2.7, this will confuse people.

Reply:
Oh, sorry. I'm misunderstanding about this. I modify this right now.